metadata.yaml

```
@@ -1,22 +1,27 @@
-# This file populates the Overview on Charmhub.
-
-# The charm package name, no spaces (required)
-# See https://juju.is/docs/sdk/naming#heading--naming-charms for guidance.
+# Copyright 2023 Canonical, Ltd.
+# See LICENSE file for licensing details.
 name: kubernetes-worker
-
-# The following metadata are human-readable and will be published prominently on Charmhub.
-
-# (Recommended)
-display-name: Charm Template
-
-# (Required)
-summary: A very short one-line summary of the charm.
-
+display-name: Kubernetes Worker
+summary: The workload bearing units of a Kubernetes cluster.
 description: |
-  A single sentence that says what the charm is, concisely and memorably.
-
-  A paragraph of one to three short sentences, that describe what the charm does.
-
-  A third paragraph that explains what need the charm meets.
-
-  Finally, a paragraph that describes whom the charm is useful for.
+  Kubernetes is an open-source platform for deploying, scaling, and operations
+  of application containers across a cluster of hosts. Kubernetes is portable
+  in that it works with public, private, and hybrid clouds. Extensible through
+  a pluggable infrastructure. Self healing in that it will automatically
+  restart and place containers on healthy nodes if a node ever goes away.
+series:
```

i wonder where the `docs` field went:

```suggestion
docs: https://discourse.charmhub.io/t/kubernetes-worker-docs-index/6104
series:
```